hellohilt/README.md

```diff
@@ -0,0 +1,91 @@
+# Dagger Usage Sample for MvRx
+
+This module contains a sample app demonstrating how to setup Hilt and AssistedInject in an app using MvRx.
+
+**NOTE** Hilts bundled implementation of AssistedInject is as time of writing not released.
+So, you will need to use the Snapshot release until Dagger 2.x is released with AssistedInject.
+
+// build.gradle
+buildscript {
+    repositories {
+        google()
+        jcenter()
+        maven { url "https://oss.sonatype.org/content/repositories/snapshots" }
+    }
+    dependencies {
+        classpath "com.google.dagger:hilt-android-gradle-plugin:HEAD-SNAPSHOT"
+        // rest of classpath(s)…
+    }
+}
+
+// module/build.gradle
+```groovy
+repositories {
```

Comment:
I believe putting this in the top level build.gradle file like:
```
allprojects {
    repositories {
        ...
    }
}
```
would be more standard